src/pages/components/chrome.mdx

```diff
@@ -170,23 +172,29 @@ Chrome uses [26px sized icons](/design/icons/chrome-icons) made specifically for
 
 <PropSheet components={props.data.mdx.components} componentName="nav" />
 
-### NavItem
+#### Nav.Item
+
+<Component components={props.data.mdx.subcomponents} componentName="nav.item" />
+
+<PropSheet components={props.data.mdx.subcomponents} componentName="nav.item" />
+
+#### Nav.ItemIcon
 
-<Component components={props.data.mdx.components} componentName="navItem" />
+<Component components={props.data.mdx.subcomponents} componentName="nav.itemicon" />
 
-<PropSheet components={props.data.mdx.components} componentName="navItem" />
+<PropSheet components={props.data.mdx.subcomponents} componentName="nav.itemicon" />
 
-### NavItemIcon
+#### Nav.ItemText
 
-<Component components={props.data.mdx.components} componentName="navItemIcon" />
+<Component components={props.data.mdx.subcomponents} componentName="nav.itemtext" />
 
-<PropSheet components={props.data.mdx.components} componentName="navItemIcon" />
+<PropSheet components={props.data.mdx.subcomponents} componentName="nav.itemtext" />
 
-### NavItemText
+#### Nav.List
 
-<Component components={props.data.mdx.components} componentName="navItemText" />
+<Component components={props.data.mdx.subcomponents} componentName="nav" />
```

Comment:
Looks like https://github.com/zendeskgarden/react-components/pull/1883 missed one. 😢 

TODO: Update to `nav.list` once fixed.